app/javascript/packages/document-capture/context/acuant.tsx

```diff
@@ -18,7 +18,6 @@ declare global {
     REPEAT_FAIL_CODE: string;
     SEQUENCE_BREAK_CODE: string;
     start?: AcuantWorkersInitialize;
```

Comment:
Should this be made non-optional now?

```suggestion
    start: AcuantWorkersInitialize;
```

Reply:
Works for me. 👍🏻